xlml/apis/test_config.py

```diff
@@ -135,7 +135,6 @@ class TestConfig(abc.ABC, Generic[A]):
   @abc.abstractmethod
   def benchmark_id(self) -> str:
     """Unique key for metrics generated by this test."""
-    ...
```

Comment:
I assume pylint was complaining about an unused expression. Should this be `pass` or `raise NotImplementedError()` instead?

Reply:
I will prefer `raise NotImplementedError()` here, especially for benchmark_id. Will update it accordingly.